### PR TITLE
socket bridge - additional check for token price

### LINKED ISSUE #1351: Problems with socket bridge
**Description**
Socket bridge stopped working

#### Definition of done
- [ ] Description is created (Task creator)
- [ ] Code is complete (Developer)
- [ ] Scope of test and what was done section is defined in task (Developer)
- [ ] Review passed (Code reviewer)
- [ ] Verified manually (QA)
- [ ] Test cases created/updated if necessary (QA)
- [ ] Change successfully merged
- [ ] Release notes added if needed

**Screenshots**

### PR DESCRIPTION
**Pull request by mix1o**
# Summary
Fixes #1351 

# To Test
- [ ] - Go to `Bridge` page
- [ ] - Select chains and token
- [ ] - Enter amount (NOTE: do not enter small amounts)
- [ ] - `Socket` should be visible

### CODE REVIEW

**Review by karczuRF — APPROVED** [review]
Good job!

**Review by mwiniarsk — APPROVED** [review]

File: src/services/EcoBridge/Socket/Socket.utils.ts [hunk 1]
```diff
@@ -35,7 +35,8 @@ const OVERRIDES = [
 ]
 
 export const getBestRoute = (routes: Route[], tokenData?: TokenPriceResponseDTO, toTokenDecimals?: number) => {
-  if (routes.length === 1 || !tokenData || !toTokenDecimals) return routes[0]
+  if (routes.length === 1 || !tokenData || !toTokenDecimals || typeof tokenData.result.tokenPrice === 'object')
```
Comment (berteotti) [inline src/services/EcoBridge/Socket/Socket.utils.ts:38]: Just to be clear, what type should `tokenPrice` be? In this case we are allowing values as `string`, `number` or `undefined`, etc. Maybe we should verify if it's the type we are expecting. This is just a suggestion 😄

**Review by berteotti — COMMENTED** [review]
Looks good, just have a small comment.

File: src/services/EcoBridge/Socket/Socket.utils.ts (hunk 1 above)
Comment (mix1o) [reply src/services/EcoBridge/Socket/Socket.utils.ts:38]: `tokenPrice` should be a `number` so I think I can change it to `typeof tokenData.result.tokenPrice !== "number"`.
Good catch mate :)

**Review by berteotti — APPROVED** [review]

File: src/services/EcoBridge/Socket/Socket.utils.ts (hunk 1 above)
Comment (mix1o) [reply src/services/EcoBridge/Socket/Socket.utils.ts]: fixed [e95336f](https://github.com/levelkdev/swapr-dapp/pull/1352/commits/e95336feef33ff0e90c698b605cc9d077591267e)

**Pull request merged**